docs/release_notes/release_notes-0.14.0.rst

```diff
@@ -44,6 +44,7 @@ Key Features and Updates
   * REFACTOR-#4206: add assert check into `__init__` method of `PandasOnDaskDataframePartition` class (#4207)
   * REFACTOR-#3900: add flake8-no-implicit-concat plugin and refactor flake8 error codes (#3901)
   * REFACTOR-#4093: Refactor base to be smaller (#4220)
+  * REFACTOR-#4284: use variable length unpacking when getting results from `deploy` function (#4285)
```

Comment:
This should be moved in `docs/release_notes/release_notes-0.15.0.rst`.

Reply:
@anmyachev, when adding this in the file please add also your GH nickname in `Contributors` list in there.

Reply:
https://github.com/modin-project/modin/pull/4403

Reply:
@anmyachev, thanks!